torat_client/netclient.go

```diff
@@ -36,7 +37,18 @@ func NetClient() {
 	log.Println("NetClient")
 	initServer()
 	var conf tor.StartConf
-	conf = tor.StartConf{ProcessCreator: embedded.NewCreator()}
+
+	tmp_dir, err := ioutil.TempDir("", "")
```

Comment:
please use camel case

Reply:
Sorry it's a force of habit Unix thing

Reply:
I see some more non camel cases I'm gonna fix